### ISSUE TITLE
USWDS should not use <abbr> tag as an accessibility pattern with form templates

### ISSUE THREAD

**Issue by jameslpetersen** [uswds/uswds#5539]
## Description

[The USWDS form templates page](https://designsystem.digital.gov/templates/form-templates/) uses the `<abbr>` tag with a `title` attribute as an accessibility pattern. This pattern doesn't actually work with the majority of browsers and screen readers, since most browsers don't support the `<abbr>` tag. The pattern only works on macOS. In a post about calendar tables, Adrian Roselli, an accessibility researcher, discusses the [lack of `<abbr>` support](https://adrianroselli.com/2022/08/brief-note-on-calendar-tables.html#Abbr-el). The same post demonstrates an [alternative technique](https://adrianroselli.com/2022/08/brief-note-on-calendar-tables.html#Hidden).

The form templates page has the following example... 
```
“A red asterisk (<abbr title="required" class="usa-hint--required">*</abbr>) indicates a required field.”
```
The alternative below would work better with most screen readers.
```
“A red asterisk (<span aria-hidden="true" class="usa-hint--required">*</span><span class="usa-sr-only">required</span>) indicates a required field.”
``` 

## Steps to reproduce the issue

1. Use Chrome/Edge/Firefox on Windows with any screen reader and listen to how the screen reader handles an `<abbr>` tag with title attribute. I tested with Windows/Chrome/NVDA.
2. The screen reader will read the contents of the `<abbr>` tag, not the contents of the title attribute.

## Additional information [optional]

* [Support matrix from PowerMapper](https://www.powermapper.com/tests/screen-readers/labelling/acronym-abbr-title/)

**Comment by thisisdano** [comment]
A good additional question for us is: Does the `abbr` functionality add meaningful value for a screen reader user?

**Comment by jameslpetersen** [comment]
Adrian Roselli posted [test results on the accessibility of the `<abbr>` element](https://adrianroselli.com/2024/01/using-abbr-element-with-title-attribute.html). It's pretty bad. There's a lot of inconsistency among combinations of browsers and assistive technologies. The answer to the question above from @thisisdano seems to be no, `<abbr>` doesn't add value to screen reader users.

**Comment by RSD-accessboard** [comment]
Following up on this, some issues with Chrome – 
<abbr>element has the * in the inner text which is read by NVDA, but it doesn't read the title = required. Because there is additional information in the title, it looks like the browser is automatically rendering a red dotted underline. It looks like having the "usa-form" class on a parent element cancels out this automatic underline, but in order to avoid this entirely I am going to use the <span  class="usa-sr-only" > method as mentioned above.

 ![Image](https://github.com/user-attachments/assets/0c0ec7f5-fce7-4c16-8815-26523c067302)

**Comment by mgifford** [comment]
Looking at https://github.com/search?q=repo%3Auswds%2Fuswds%20abbr&type=code

`<abbr>` is mostly used with the required field. 

We might want to get rid of this CSS at some point:
`abbr[title="required"] {`

From   packages/usa-form/src/styles/_usa-form.scss

If we have updated the form's required elements, 

We wouldn't want to remove:


```
abbr[title] {
--
text-decoration: underline;
text-decoration: underline dotted;
}
```

From packages/uswds-elements/lib/_normalize.scss as this will affect content pages where `<abbr>` may already be defined. 

But we should look at updating the form element's required (or optional) fields. The Drupal community has gone through looking at how to do this a number of times/ways over he years. This is the latest approach:
https://www.drupal.org/project/drupal/issues/2921627

Worth looking at:
- https://www.w3.org/WAI/test-evaluate/easy-checks/required-fields/
- https://vispero.com/resources/doing-whats-required-indicating-mandatory-fields-in-an-accessible-way/
- https://www.digitala11y.com/anatomy-of-creating-accessible-forms-is-this-field-required/
- https://dequeuniversity.com/tips/identify-required-elements
- https://www.deque.com/blog/anatomy-of-accessible-forms-required-form-fields/
- https://forms.sis.yale.edu/designer-resources/best-practices
- https://accessibility.huit.harvard.edu/technique-required-fields

**Comment by chandracarney** [comment]
In June, we discussed that this needs further testing and triage. Since this is an older issue, here are some things we discussed:
- Needs some research; maybe a better approach these days 
- Might need another look at where we're using it
- https://caniuse.com/?search=abbr
